app/serializers/preprint-provider.ts

```diff
@@ -1,9 +1,10 @@
 import OsfSerializer from './osf-serializer';
 
-export default class PreprintProvider extends OsfSerializer {}
+export default class PreprintProviderSerializer extends OsfSerializer {
+}
 
-declare module 'ember-data' {
-    interface SerializerRegistry {
-        'preprint-provider': PreprintProvider;
-    }
+declare module 'ember-data/types/registries/serializer' {
+    export default interface SerializerRegistry {
+        'preprint-provider': PreprintProviderSerializer;
```

Comment:
This is the answer to my question above, yes?

Reply:
The above was for `preprint`, but yeah, it uses the registry to look up the serializer class.